Studen A - Tasks

ramiaquista opened the pull request:
## What I have done 🔨 
- Added mealAPI.js file to fetch a random meal from the API.
- Added styling to list-items container, recipe-container, item-container.
- Removed link reference to style.css file inside index.html head.
- Added background img to the body in style.css file.
- Added Recipes counter that is displayed on the navBar.
- Added counterItem.test.js file
- Added correct count of recipes testing situation with 7 tests.
- Added incorrect count of recipes testing situation with 3 tests.
- Removed old APP_ID const form index.js file.
- Added styling to cards in the style.css file

ramiaquista committed:
Add listItems.js file with displayList async().
Add mealAPI.js file to fetch a random meal from the API.
Add styling to list-items container, recipe-container, item-container.
Remove link reference to style.css file inside index.html head.

SunnySparks commented:
In general, the project is going well, we are on time for complying with the requirements, keep it up 👍🏾

ramiaquista committed:
Add involvementAPI.js file with giveLike() and refreshItemLikes().
Add refreshLikes() function inside index.js file.
Add eventListener to HeartIcon so when it's pressed it call giveLike().
Add loadStorage.js file with getLocalStorage() and setLocalStorage().

ramiaquista committed:
Change of DOMContentLoad logic to display items while fetch the like's API.
Add bakcground img to body in style.css file.
Change color of recipes cards in style.css file.
Change font color of recipes cards.
Add Recipes counter that is displayed on the navBar.

ramiaquista committed:
Add counterItem.js file with counter() function to keep tracking of how many elements are displayed.
Add counterItem.test.js file
Add correct count of recipes testing situation with 7 test's.
Add uncorrect count of recipes testing situation with 3 test's.
Remove old APP_ID const form index.js file.

ramiaquista committed:
Add styling to cards in style.css file

ramiaquista committed:
Add logic to remove items when the user change between page sections.
Add comment btn eventListener capturing the id of the food that is being pressed.
Add d-none class inside style.css file.

ramiaquista committed:
Fix linters erorrs inside index.js file

ramiaquista committed:
Change logic of display random amount of foods

SunnySparks commented:
## Status: Approved 🎉

### Congratulations @ramiaquista

you've been doing an amazing job so far and polished your code properly for the app to work. Keep it up and happy coding 😄
